app/Platform/EventServiceProvider.php

```diff
@@ -101,6 +103,7 @@ class EventServiceProvider extends ServiceProvider
             DispatchUpdateGameMetricsJob::class, // dispatches GameMetricsUpdated
         ],
         PlayerMetricsUpdated::class => [
+            DispatchUpdatePlayerPointsStatsJob::class, // dispatches PlayerPointsStatsUpdated
```

Comment:
This doesn't seem to be working for me. If I curl a dorequest to unlock an achievement, my Daily stats don't get updated. I did `sail artisan horizon:terminate` and `sail artisan horizon` to restart the service, and I see the "UpdatePlayerMetrics" job go through Horizon, but not this. When I tried resetting my VM (to ensure Horizon was restarted), my database became empty.

Reply:
Do you see a similar result when you use the admin tool to manually unlock an achievement for a user? This is what I see:

![Screenshot 2024-06-03 at 8 08 55 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/5410e192-6be1-4468-bc4d-b985134f72ae)

Reply:
With latest, the player points job is getting run, and the points show up if I click the more link:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/2ec6a448-9403-4b8b-b63d-aa7d8af3ee12)

But they're still not present on the profile page itself:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/d0d65347-f671-43b5-bf45-61da9120dce6)

I've verified the rows are not present in the database, which implies the queries for the more page are not using the denormalized data.

If I run the job manually: `sail artisan ra:platform:player:update-points-stats Jamiras`, then the rows do get created, but the player still doesn't show up in the daily/weekly tabs on the profile page.

Reply:
It looks like it's just not including the current player on the daily/weekly tabs. If I manually unlock an achievement for a followed user, and manually run their associated update-points-stats job, _they_ appear in the daily/weekly tabs:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/2ba518f0-daa4-4f20-a1f5-165cfd42fed9)
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/900a2570-c9d9-4c40-8d64-80b2d88acec1)

Reply:
Fixed a few odds and ends in latest. I believe I hunted down why the job wasn't dispatching when doing a curl to the Connect API with `r=awardachievement`. I've also made adjustments to include the current user in the Daily and Weekly view, as well as fixed a bug with the rank # for the current user.

![Screenshot 2024-06-05 at 7 57 45 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/3d60d279-e1c5-40d2-a0f7-85c297129856)